celery/contrib/django/task.py

```diff
@@ -12,10 +12,10 @@ class DjangoTask(Task):
     Provide a nicer API to trigger tasks at the end of the DB transaction.
     """
 
-    def delay_on_commit(self, *args, **kwargs):
+    def delay_on_commit(self, *args, **kwargs) -> None:
```

Comment:
Not sure what is your policy on type hints, I'm hoping that this might reinforce this fact and perhaps help code editor to spot issues.